lib/WebpackConfig.js

```diff
@@ -725,6 +734,18 @@ class WebpackConfig {
         this.useImagesLoader = false;
     }
 
+    enableImagemin(options = {}) {
+        this.useImagemin = true;
+
+        for (const optionKey of Object.keys(options)) {
+            if (!(optionKey in this.imageminOptions)) {
+                throw new Error(`Invalid option "${optionKey}" passed to enableImagemin(). Valid keys are ${Object.keys(this.imageminOptions).join(', ')}`);
+            }
+
+            this.imageminOptions[optionKey] = options[optionKey];
+        }
```

Comment:
That kind of validation is usually only done for Encore-specific options (the reason being that it would, for instance, disallow the valid `plugins` option in this case).

Reply:
so we shouldn't validate the configuration at all, right?

Reply:
> so we shouldn't validate the configuration at all, right?

Exactly